### PR TITLE
Style fixes for filters, metrics_math, and metrics_math_test

### PR DESCRIPTION
**Pull request by mtlynch**

### CODE REVIEW

File: telescope/filters.py [hunk 1]
```diff
@@ -82,212 +91,214 @@ def _filter_c2s_measurement(measurement):
...
+           int(measurement['web100_log_entry_snap_SndLimTimeCwnd']) +
+           int(measurement['web100_log_entry_snap_SndLimTimeSnd']))
+          >= _MINIMUM_DURATION):
     return False
   if not ((int(measurement['web100_log_entry_snap_SndLimTimeRwin']) +
-                            int(measurement['web100_log_entry_snap_SndLimTimeCwnd']) +
-                            int(measurement['web100_log_entry_snap_SndLimTimeSnd'])) < _MAXIMUM_DURATION):
+           int(measurement['web100_log_entry_snap_SndLimTimeCwnd']) +
+           int(measurement['web100_log_entry_snap_SndLimTimeSnd']))
+          < _MAXIMUM_DURATION):
```
Comment (pboothe) [inline telescope/filters.py]: total_time = (int(measurement['web100_log_entry_snap_SndLimTimeRwin']) +  int(measurement['web100_log_entry_snap_SndLimTimeCwnd']) + int(measurement['web100_log_entry_snap_SndLimTimeSnd']))

if not (_MINIMUM_DURATION <= total_time < _MAXIMUM_DURATION):
  return False

Comment (pboothe) [inline telescope/filters.py]: or even

return ((_MINIMUM_DURATION <= total_time < _MAXIMUM_DURATION) and (int(measurement['web100_log_entry_snap_HCThruOctetsAcked']) >= _MINIMUM_PACKETS)  and (int(measurement['web100_log_entry_snap_CongSignals']) > 0) and _filter_invalid_tcp_state(measurement))

**Comment by pboothe** [comment]
Nitpicks for more cleanup, but LGTM

File: telescope/filters.py (hunk 1 above)
Comment (collina) [inline telescope/filters.py]: I would do the first for readability.

Comment (mtlynch) [inline telescope/filters.py]: Fixed

**Comment by mtlynch** [comment]
Thanks for the review!

**Pull request merged**